src/otx/algorithms/common/adapters/mmcv/nncf/runners.py

```diff
@@ -111,7 +111,11 @@ def dump_checkpoint_fn(model, compression_ctrl, nncf_runner, save_dir):
             self._eval_hook._save_ckpt(self, nncf_runner.best_val_metric_value)
             return self._eval_hook.best_ckpt_path
 
-        uncompressed_model_accuracy = self.model.module.nncf._uncompressed_model_accuracy
+        if hasattr(self.model, "module"):
```

Comment:
It's interesting, if we use only CPU then `self.model` doesn't contain the `module`?